src/uistate.h

```diff
@@ -161,7 +161,7 @@ class uistatedata
         bool distraction_withdrawal = true;
         bool distraction_all = true;
         bool numpad_navigation = false;
-        bool is_toggle = false;
+        bool is_toggle = false; // NOLINTNEXTLINE(cata-serialize)
```

Comment:
It seems to work fine, if only that toggling noise now makes the "toggle all" (graphically) change with it. It's fine though, since there's nothing beyond that it looks off.

The solution you propose is agreeable.

Reply:
I ended up going back on this. The visual oddity is undesirable. Your point about not needing `uistate.is_toggle` is great and I still went with it.